tests/e2e/init-node.sh

```diff
@@ -97,11 +97,14 @@ evmosd gentx $KEY 1000000000000000000000aevmos --keyring-backend $KEYRING --chai
 ## 4. Run `gentx` in each of those folders
 ## 5. Copy the `gentx-*` folders under `~/.clonedEvmosd/config/gentx/` folders into the original `~/.evmosd/config/gentx`
 
+# Enable the APIs for the tests to be successful
+sed -i 's/enable = false/enable = true/g' "$APP_TOML"
+
 # Collect genesis tx
 evmosd collect-gentxs
 
 # Run this to ensure everything worked and that the genesis file is setup correctly
 evmosd validate-genesis
 
 # Start the node (remove the --pruning=nothing flag if historical queries are not needed)
```

Comment:
Probably we should update also this comment since `--pruning=nothing` is not present in the command.

Reply:
good catch!